packages/sampler/.storybook/manager.js

```diff
@@ -8,8 +9,8 @@ addons.setConfig({
 		brandUrl: 'http://enactjs.com/',
 		brandImage: 'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI3MDAiIGhlaWdo  dD0iNzAwIiB2aWV3Qm94PSIwIDAgNzAwIDcwMCI+PGRlZnM+PHN0eWxlPi5hLC5ie2ZpbGw6I2ZmZjt9LmJ7b3BhY2l0eTowLjg7fTwvc3R5bGU+PC9kZWZzPjxwb2x  5Z29uIGNsYXNzPSJhIiBwb2ludHM9IjM0OS45IDUyNy44IDE5OS45IDQyOS44IDE5OS45IDM3NC40IDM1MC4xIDQ3Mi43IDM0OS45IDM0OC41IDE5OS45IDI2MS4xID  E5OS45IDIwOS4zIDM1MCAyOTMuNiAzNTAgMjkzLjYgMzUwLjIgMjkzLjcgMzQ5LjkgMTY5LjMgMTAyLjcgNDguNSAxMDIuNyA0NzIuNiAzNTAuMSA2NTEuNiAzNDkuO  SA1MjcuOCIvPjxwb2x5Z29uIGNsYXNzPSJiIiBwb2ludHM9IjM1MC4xIDY1MS42IDU5Ny4zIDQ3Mi44IDU5Ny4zIDM2Ni4zIDM0OS45IDUyNy44IDM1MC4xIDY1MS42  Ii8+PHBvbHlnb24gY2xhc3M9ImIiIHBvaW50cz0iMzUwLjEgNDcyLjcgNTIzLjUgMzU5LjMgNTIzLjUgMjQ3LjQgMzQ5LjkgMzQ4LjUgMzUwLjEgNDcyLjciLz48cG9  seWdvbiBjbGFzcz0iYiIgcG9pbnRzPSIzNTAgMjkzLjYgMzUwIDI5My42IDM1MC4yIDI5My43IDU5Ny4zIDE1NC44IDU5Ny4zIDQ4LjQgMzQ5LjkgMTY5LjMgMzUwID  I5My42Ii8+PC9zdmc+'
 	}),
-	isFullscreen: false,
-	showNav: true,
-	showPanel: true,
-	panelPosition: 'bottom'
+	panelPosition: 'bottom',
+	bottomPanelHeight: ri.scaleToRem(300),
+	// rightPanelWidth: ri.scaleToRem(150),
+	// navSize: ri.scaleToRem(150)
```

Comment:
Do we need this properties?

Reply:
sorry, i pushed the commented properties by mistake. i will  revert to the rpevious value

Reply:
No, I think these 3 properties were introduced in this PR so I'd want to know that for what reason you've added these properties.

Reply:
In case you've missed this comment @daniel-stoian-lgp

Reply:
ah, i misunderstood your comment. here is the source: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#ui-layout-state-has-changed-shape 
and api documentation https://storybook.js.org/docs/8.0/configure/features-and-behavior

Reply:
How about removing these properties then?

Reply:
sounds good to me